modin/pandas/base.py

```diff
@@ -162,7 +162,7 @@ def _get_repr_axis_label_indexer(labels, num_for_repr):
 
 
 @_inherit_docstrings(pandas.DataFrame, apilink=["pandas.DataFrame", "pandas.Series"])
-class BasePandasDataset(ClassLogger):
+class BasePandasDataset(ClassLogger, modin_layer="DATASET"):
```

Comment:
```suggestion
class BasePandasDataset(ClassLogger):
```
PANDAS-API should be here, which is default.